### PR TITLE
Correct OA requirements

### PR DESCRIPTION
**Pull request by wsjsxn**
Correct OA requirements. The google spreadsheet hasn't been updated and will be updated after the pull request is approved.

### CODE REVIEW

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 1]
```diff
@@ -6872,8 +6872,8 @@
       "ventilation_standard": "ASHRAE 62.1-1999",
       "ventilation_primary_space_type": "Hotels, Motels, Resorts, Dormitories",
       "ventilation_secondary_space_type": "Dormitory sleeping areas",
-      "ventilation_per_area": 0.0,
-      "ventilation_per_person": 15.0,
+      "ventilation_per_area": 0.142855357,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:6875]: Based on 30 cfm/room for a two room suite

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 2]
```diff
@@ -7048,8 +7048,8 @@
       "ventilation_standard": "ASHRAE 62.1-2007",
       "ventilation_primary_space_type": "Hotels, Motels, Resorts, Dormitories",
       "ventilation_secondary_space_type": "Bedroom/living Room",
-      "ventilation_per_area": 0.06,
-      "ventilation_per_person": 5.0,
+      "ventilation_per_area": 0.142855357,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:7051]: Based on 30 cfm/room for a two room suite

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 3]
```diff
@@ -6960,8 +6960,8 @@
       "ventilation_standard": "ASHRAE 62.1-2007",
       "ventilation_primary_space_type": "Hotels, Motels, Resorts, Dormitories",
       "ventilation_secondary_space_type": "Bedroom/living Room",
-      "ventilation_per_area": 0.06,
-      "ventilation_per_person": 5.0,
+      "ventilation_per_area": 0.132557114,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:6963]: Based on 35 cfm/bath

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 4]
```diff
@@ -7488,8 +7488,8 @@
       "ventilation_standard": "ASHRAE 62.1-2007",
       "ventilation_primary_space_type": "Hotels, Motels, Resorts, Dormitories",
       "ventilation_secondary_space_type": "Bedroom/living Room",
-      "ventilation_per_area": 0.06,
-      "ventilation_per_person": 5.0,
+      "ventilation_per_area": 0.132611175,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:7491]: Based on 35 cfm/bath

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 5]
```diff
@@ -16352,7 +16352,7 @@
       "ventilation_primary_space_type": "Education",
       "ventilation_secondary_space_type": "Classroom",
       "ventilation_per_area": 0.0,
-      "ventilation_per_person": 16.95,
+      "ventilation_per_person": 15.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:16355]: The value is wrong and the new value comes from 62.1.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 6]
```diff
@@ -8012,11 +8012,11 @@
...
-      "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "Retail",
-      "ventilation_secondary_space_type": "General Sales",
-      "ventilation_per_area": 0.12,
-      "ventilation_per_person": 7.5,
+      "ventilation_standard": "ASHRAE 62.1-1999",
+      "ventilation_primary_space_type": "Retail Stores, Sales Floors, and Show Room Floors",
+      "ventilation_secondary_space_type": "Basement and street",
+      "ventilation_per_area": 0.3,
+      "ventilation_per_person": 0.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:8019]: The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 7]
```diff
@@ -4148,7 +4148,7 @@
       "ventilation_standard": "ASHRAE 62.1-2004",
       "ventilation_primary_space_type": "Basement",
       "ventilation_secondary_space_type": "For Hospital",
-      "ventilation_per_area": 0.049,
+      "ventilation_per_area": 0.05,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:4151]: The value was updated based on 62.1.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 8]
```diff
@@ -6782,9 +6782,9 @@
       "occupancy_activity_schedule": "HotelLarge ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "General",
-      "ventilation_secondary_space_type": "Corridors",
-      "ventilation_per_area": 0.06,
+      "ventilation_primary_space_type": "Public Spaces",
+      "ventilation_secondary_space_type": "Corridors and utilities",
+      "ventilation_per_area": 0.05,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:6787]: The primary and secondary space types were corrected and the ventilation rates were updated accordingly.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json (hunk 5 above)
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:16355]: The value is wrong and the new value was added based on 62.1

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 9]
```diff
@@ -16440,7 +16440,7 @@
       "ventilation_primary_space_type": "Education",
       "ventilation_secondary_space_type": "Classroom",
       "ventilation_per_area": 0.0,
-      "ventilation_per_person": 16.95,
+      "ventilation_per_person": 15.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:16443]: The value is wrong and the new value was added based on 62.1

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 10]
```diff
@@ -16792,7 +16792,7 @@
       "ventilation_primary_space_type": "Education",
       "ventilation_secondary_space_type": "Libraries",
       "ventilation_per_area": 0.0,
-      "ventilation_per_person": 16.95,
+      "ventilation_per_person": 15.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:16795]: The value is wrong and the new value was added based on 62.1

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 11]
```diff
@@ -18024,7 +18024,7 @@
       "ventilation_primary_space_type": "Education",
       "ventilation_secondary_space_type": "Classroom",
       "ventilation_per_area": 0.0,
-      "ventilation_per_person": 16.95,
+      "ventilation_per_person": 15.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:18027]: The value is wrong and the new value was added based on 62.1

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 12]
```diff
@@ -18112,7 +18112,7 @@
...
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:18115]: The value is wrong and the new value was added based on 62.1

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 13]
```diff
@@ -18464,7 +18464,7 @@
       "ventilation_primary_space_type": "Education",
       "ventilation_secondary_space_type": "Libraries",
       "ventilation_per_area": 0.0,
-      "ventilation_per_person": 16.95,
+      "ventilation_per_person": 15.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:18467]: The value is wrong and the new value was added based on 62.1

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json [hunk 14]
```diff
@@ -18549,10 +18549,10 @@
...
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-1999",
-      "ventilation_primary_space_type": "Education",
-      "ventilation_secondary_space_type": "Corridors",
-      "ventilation_per_area": 0.1,
-      "ventilation_per_person": 0.0,
+      "ventilation_primary_space_type": "Offices",
+      "ventilation_secondary_space_type": "Reception Areas",
+      "ventilation_per_area": null,
+      "ventilation_per_person": 15.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json:18555]: The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/data/ashrae_90_1_2007.spc_typ.json [hunk 15]
```diff
@@ -13291,10 +13291,10 @@
       "occupancy_schedule": "OfficeSmall BLDG_OCC_SCH",
       "occupancy_activity_schedule": "OfficeSmall ACTIVITY_SCH",
       "is_residential": null,
-      "ventilation_standard": "ASHRAE 62.1-1999",
+      "ventilation_standard": "ASHRAE 62.1-2004",
       "ventilation_primary_space_type": "Office Buildings",
       "ventilation_secondary_space_type": "Whole Building",
-      "ventilation_per_area": 0.1,
+      "ventilation_per_area": 0.085,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/data/ashrae_90_1_2007.spc_typ.json:13297]: The ventilation standard was revised and the ventilation rates were updated accordingly.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/data/ashrae_90_1_2007.spc_typ.json [hunk 16]
```diff
@@ -18484,10 +18484,10 @@
       "occupancy_activity_schedule": "SchoolSecondary ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-2004",
-      "ventilation_primary_space_type": "General",
-      "ventilation_secondary_space_type": "Corridors",
+      "ventilation_primary_space_type": "Office",
+      "ventilation_secondary_space_type": "Buildings Main entry lobbies",
       "ventilation_per_area": 0.06,
-      "ventilation_per_person": null,
+      "ventilation_per_person": 5.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/data/ashrae_90_1_2007.spc_typ.json:18490]: The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2010/data/ashrae_90_1_2010.spc_typ.json [hunk 17]
```diff
@@ -18659,10 +18659,10 @@
       "occupancy_activity_schedule": "SchoolSecondary ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "General",
-      "ventilation_secondary_space_type": "Corridors",
+      "ventilation_primary_space_type": "Office",
+      "ventilation_secondary_space_type": "Buildings Main entry lobbies",
       "ventilation_per_area": 0.06,
-      "ventilation_per_person": null,
+      "ventilation_per_person": 5.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2010/data/ashrae_90_1_2010.spc_typ.json:18665]: The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2013/data/ashrae_90_1_2013.spc_typ.json [hunk 18]
```diff
@@ -5201,10 +5201,10 @@
       "occupancy_schedule": "Hospital BLDG_OCC_EXTD_SCH",
       "occupancy_activity_schedule": "Hospital ACTIVITY_SCH",
       "is_residential": null,
-      "ventilation_standard": "ASHRAE 62.1-2007",
+      "ventilation_standard": "ASHRAE 62.1-2010",
       "ventilation_primary_space_type": "Food and Beverage Service",
-      "ventilation_secondary_space_type": "Cafeteria/fast food dining",
-      "ventilation_per_area": 0.18,
+      "ventilation_secondary_space_type": "Kitchen (cooking)",
+      "ventilation_per_area": 0.12,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2013/data/ashrae_90_1_2013.spc_typ.json:5207]: The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2013/data/ashrae_90_1_2013.spc_typ.json [hunk 19]
```diff
@@ -18955,10 +18955,10 @@
       "occupancy_activity_schedule": "SchoolSecondary ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "General",
-      "ventilation_secondary_space_type": "Corridors",
+      "ventilation_primary_space_type": "Office",
+      "ventilation_secondary_space_type": "Buildings Main entry lobbies",
       "ventilation_per_area": 0.06,
-      "ventilation_per_person": null,
+      "ventilation_per_person": 5.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2013/data/ashrae_90_1_2013.spc_typ.json:18961]: The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2016/data/ashrae_90_1_2016.spc_typ.json [hunk 20]
```diff
@@ -5201,10 +5201,10 @@
       "occupancy_schedule": "Hospital BLDG_OCC_EXTD_SCH",
       "occupancy_activity_schedule": "Hospital ACTIVITY_SCH",
       "is_residential": null,
-      "ventilation_standard": "ASHRAE 62.1-2007",
+      "ventilation_standard": "ASHRAE 62.1-2013",
       "ventilation_primary_space_type": "Food and Beverage Service",
-      "ventilation_secondary_space_type": "Cafeteria/fast food dining",
-      "ventilation_per_area": 0.18,
+      "ventilation_secondary_space_type": "Kitchen (cooking)",
+      "ventilation_per_area": 0.12,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2016/data/ashrae_90_1_2016.spc_typ.json:5207]: The ventilation secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2016/data/ashrae_90_1_2016.spc_typ.json [hunk 21]
```diff
@@ -18925,10 +18925,10 @@
       "occupancy_activity_schedule": "SchoolSecondary ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "General",
-      "ventilation_secondary_space_type": "Corridors",
+      "ventilation_primary_space_type": "Office",
+      "ventilation_secondary_space_type": "Buildings Main entry lobbies",
       "ventilation_per_area": 0.06,
-      "ventilation_per_person": null,
+      "ventilation_per_person": 5.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2016/data/ashrae_90_1_2016.spc_typ.json:18931]: The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.spc_typ.json [hunk 22]
```diff
@@ -5201,10 +5201,10 @@
       "occupancy_schedule": "Hospital BLDG_OCC_EXTD_SCH",
       "occupancy_activity_schedule": "Hospital ACTIVITY_SCH",
       "is_residential": null,
-      "ventilation_standard": "ASHRAE 62.1-2007",
+      "ventilation_standard": "ASHRAE 62.1-2016",
       "ventilation_primary_space_type": "Food and Beverage Service",
-      "ventilation_secondary_space_type": "Cafeteria/fast food dining",
-      "ventilation_per_area": 0.18,
+      "ventilation_secondary_space_type": "Kitchen (cooking)",
+      "ventilation_per_area": 0.12,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.spc_typ.json:5207]: The ventilation standard and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.spc_typ.json [hunk 23]
```diff
@@ -18925,10 +18925,10 @@
       "occupancy_activity_schedule": "SchoolSecondary ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "General",
-      "ventilation_secondary_space_type": "Corridors",
+      "ventilation_primary_space_type": "Office",
+      "ventilation_secondary_space_type": "Buildings Main entry lobbies",
       "ventilation_per_area": 0.06,
-      "ventilation_per_person": null,
+      "ventilation_per_person": 5.0,
```
Comment (wsjsxn) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.spc_typ.json:18931]: The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.

**Review by mdahlhausen — APPROVED** [review]
These updates look fine.

- Please regenerate the regression tests for the affected buildings
- There was an empty 'git' file added to the pull request.  Please remove it.

**Review by mdahlhausen — CHANGES_REQUESTED** [review]
changing to request changes until regression tests are updated

**Comment by wsjsxn** [comment]
> These updates look fine.
> 
> * Please regenerate the regression tests for the affected buildings
> * There was an empty 'git' file added to the pull request.  Please remove it.

I have removed the empty git file and please review the changes. Thanks

**Review by mdahlhausen — APPROVED** [review]

**Pull request merged**